accord-core/src/main/java/accord/utils/async/AsyncChains.java

```diff
@@ -589,14 +589,21 @@ protected void start(BiConsumer<? super T, Throwable> callback)
 
     public static <V> AsyncChain<V> ofCallable(Executor executor, Callable<V> callable)
     {
-        return new Head<V>()
+        return ofCallable(executor, callable, AsyncChains::encapsulate);
+    }
+
+    public static <V> AsyncChain<V> ofCallable(Executor executor,
```

Comment:
why do we need to expose this method?

Reply:
It's called from `TxnNamedRead#performLocalRead()`